R/background/rmarkdown/knit.R

```diff
@@ -0,0 +1,22 @@
+# requires rmarkdown package to run (and knitr)
+if (!require(rmarkdown, quietly = TRUE)) {
```

Comment:
Should we use `requireNamespace("rmarkdown", quietly = TRUE)` here or we do want to attach the package?

Reply:
Thanks for catching that!